GameData/RealismOverhaul/Waterfall_Configs/ReStock/restock_nerva.cfg

```diff
@@ -0,0 +1,20 @@
+@PART[nuclearEngine]:NEEDS[Waterfall,ReStock]:AFTER[000_ReStock]
+{
+    @MODULE[ModuleEnginesFX]
+    {
+        @runningEffectName = running
+    }
+}
+
+
+@PART[nuclearEngine]:NEEDS[Waterfall,ReStock]:BEFORE[ROWaterfall]
+{
+    ROWaterfall
+    {
+        template = BDB_nuclear_PBR_vac
+        audio = pump-fed-light-1
+        position = 0,0,0.107
+        rotation = 0, 0, 0
+        scale = 1.23, 1.23, 1.23
+    }
+}
```

Comment:
This part is the basis for all of the Bornholio Nuclear parts.  Whatever model is applied to this part becomes for all the others, with various rescaling.  Suggest for sanity that these plume patches compress into a single file, rather than separate files for each, and this will allow you to highlight the patching difference between a Restock-provided model and a stock model.  What about VSR?

One example, you might write the base patch, then write a patch with NEEDS[ReStock] to edit, then write a patch with :NEEDS[VenStockRevamp/Squad] and do an alternate replacement.

Reply:
Sure, I can reorganize these patches into one file.

However, I still think that each model (Restock/VSR) should be patched in separate files, because from the view of Waterfall, they may as well be separate parts.

Reply:
Personally, I would prefer that they went into the same file.  Reason being all of the effort that went into deconflicting the separate stock, VSR, and Restock patches for most stock parts, highlighting a vast amount of patch duplication.  If something about the plume changes, I would much prefer to avoid, or at least make it less likely, that the change doesn't propagate to the other files.

Reply:
As someone who is very pro-VSR (but also now very pro-Waterfall), I would love to be sure that VSR models are handled appropriately. I realize that's annoying given the close links between Waterfall and ReStock, but still. :)

Reply:
VSR will 100% happen (it's on the [todo list](https://github.com/vevladdd/ROE-Waterfall/issues/5)), it's just a question of when. I'm not a VSR user myself, but it's used widely enough to warrant support.

If you made me guess, I'd say it'll happen in June.

Reply:
Great! Think the only other Ven's stuff in heavy use is the various stuff derived from the LV-1 model?